index.html

```diff
@@ -1,16 +1,177 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      <div class="header-block">
+        <img class="header-item1" src="images/logon.png" alt="Logo Image" width="" /> 
+        <div class="header-item2">Contact</div>
+      </div>  
+    
+    
+    </header>
+
+    <main>
+  <nav class="navbar">
+    <!--<img class="logo" src="images/logo.png" alt="Logo Image" width="" /> -->
+    <ul class="main-nav">
+      <li>
+        <a href="#" class="nav-links">Home</a>
+      </li>
+      <li>
+        <a href="#" class="nav-links">CAKES</a>
+      </li>
+      <li>
+        <a href="#" class="nav-links">ORDERING</a>
+      </li>
+      <li>
+        <a href="#" class="nav-links">LESSONS</a>
+      </li>
+      <li>
+        <a href="#" class="nav-links">ABOUT</a>
+      </li>
+      
+    </ul>
+  </nav>
+  <section>
+    <di 
+   </section>
+   <div class="our-story" >
+     <div class="read-more">
+       <h1 class="page_title">Grace Bakery</h1>
+       <p class="story-text">
+         Grace Bakery was founded in October 2016. Having started out baking for children's parties, quickly noticed how well the cupcakes went down with adults! <br><br> We have three shops located in London. It gives us all a huge sense of satisfaction to see people queuing out into the street on a Saturday afternoon, waiting to buy our cakes - there is no better feeling than this, to see that people really do want to buy and eat what we have made. We have tried to keep it to simple old fashioned cooking, using good quality, seasonal and well sourced ingredients, as one might use at home when baking for your own children, friends and family.
+        </p>
+        </div>
+   </div>
+   <section>
+    <h2>Featured Products</h2>
+     <div class="featured_products ">
+         
+
+         <div class="cake-images">
+           
+             <img
+               src="/images/0000518_ginger-fudge-cupcake_360.jpeg"
+               alt="Ginger Fudge Cupcake"
+             />
+             <div class="featured_products_text">
+               <h4>Ginger Fudge Cupcake</h4>
+               <h5>Price: £2.95</h5>
+               <button class="featured_products_button">Add To Cart</button>
+             </div>
+           </a>
+         </div>
+
+         <div class="cake-images">
+           
+             <img
+               src="/images/0002148_valentines-day-large-gift-box_360.jpeg"
+               alt="Valentine&#39;s Day Large Gift Box"
+             />
+             <div class="featured_products_text">
+               <h4>Valentine's Day Large Gift Box</h4>
+               <h5>Price: £39.00</h5>
+               <button class="featured_products_button">Add To Cart</button>
+             </div>
+           </a>
```

Comment:
looks like this extra closing `a` tag has been repeated for the rest of your cake images divs....